src/frontends/tensorflow/src/op/approximateequal.cpp

```diff
@@ -0,0 +1,40 @@
+// approximateequal.cpp
+#include <ngraph/node.hpp>
+#include <ngraph/opsets/opset1.hpp>
+#include <ngraph/op/util/op_types.hpp>
+#include <ngraph/validation_util.hpp>
+
+using namespace std;
+using namespace ov;
+using namespace ov::opset1;
```

Comment:
please avoid use of opset header and include concrete headers for concrete operations you are using for your decomposition below.